### PR TITLE
Cut off pr body in bors merge commit

### LINKED ISSUE #6756: Cut off PR description checklists from merge commit messages
In #6642, we've further expanded the PR descriptions. When merging, bors copies the entire PR description into the merge commit message. These messages are getting too long.

>> Since the PR description is growing, it would be nice to remove the Definition of Ready and Definition of Done sections from the final (merge) commit to avoid noise in the commit history.

> Good idea, we could use the `cut_body_after` configuration option from bors for this:

See: https://bors.tech/documentation/ (in the reference tab)

_Originally posted by @korthout in https://github.com/camunda-cloud/zeebe/issues/6642#issuecomment-815499247_

### PR DESCRIPTION
**Pull request by korthout**
## Description

This adds a specific marker to cut off the merge commit body.

The definition of ready and definition of done checklists add a lot of
unnecessary volume to merge commits, because bors copies the pr
description into the merge commit body. We already instructed bors to
cut off these bodies at a marker, but the marker has been removed at
some point. Having a specific marker for it reduces the change we remove
it by accident.

## Related issues

closes #6756

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

**Comment by korthout** [comment]
bors try

**Comment by korthout** [comment]
bors cancel

The try commit does not use this mechanism, so I don't know how we can test this before.

**Comment by korthout** [comment]
@npepinpe can you have a look at this?

**Comment by npepinpe** [comment]
If we want to be clean, we can test it on a different repository with bors enabled. If we don't care, we can just test it here directly with test PRs :slightly_smiling_face:

File: bors.toml [hunk 1]
```diff
@@ -8,4 +8,5 @@ delete_merged_branches = true
 
 commit_title = "merge: ${PR_REFS}"
 
-cut_body_after = "# Pull Request Checklist"
+# Also cuts the marker, not just after
```
Comment (npepinpe) [inline bors.toml:11]: Very useful comment, that was going to be my first question :smile:

**Review by npepinpe — APPROVED** [review]
From the docs it should work, let's just try :+1:

**Comment by korthout** [comment]
@npepinpe I agree. I think it takes less time to just do here.

bors merge

**Comment by korthout** [comment]
Note that it is not yet activated for the merge commit that merges this pr.

**Comment by npepinpe** [comment]
We can do a test PR afterwards

**Comment by korthout** [comment]
@npepinpe it worked: see 3b64e4ae0cbd772a7403139d6c30b68c3775cb69

**Pull request merged**